packages/uploadthing/test/sdk.test.ts

```diff
@@ -584,14 +592,18 @@ describe.runIf(shouldRun)(
       const result = await utapi.uploadFiles(file, {
         acl: "private",
       });
+      const key = result.data!.key;
       expect(result).toEqual({
         data: {
           customId: null,
           key: expect.stringMatching(/.+/),
           name: "foo.txt",
           size: 3,
           type: "text/plain",
-          url: expect.stringMatching(/https:\/\/utfs.io\/f\/.+/),
+          url: "https://utfs.io/f/" + key,
```

Comment:
Shouldn't this also be a `stringMatching`?

Reply:
Ah i guess it's not really necessary since there is no unknown part of the url to match.